contracts/smart-account/modules/SessionValidationModules/ERC20SessionValidationModule.sol

```diff
@@ -3,6 +3,20 @@ pragma solidity 0.8.17;
 import "./ISessionValidationModule.sol";
 import {ECDSA} from "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
 
+/**
+ * @title ERC20 Session Validation Modul for Biconomy Smart Accounts.
+ * @dev Validates ERC20 transfers (and approvals) session key signed userOps
```

Comment:
Can be better worded.

Validates userOps for ERC20 transfers and approvals using a session key signature.

Reply:
fixed thnx